### PR TITLE
GPC-NONE: Add logging for deleting cognito client and sqs queue

### PR DESCRIPTION
**Pull request by oskarwilliams**

### CODE REVIEW

File: src/main/kotlin/uk/gov/gdx/datashare/services/AcquirersService.kt [hunk 1]
```diff
@@ -177,15 +183,18 @@ class AcquirersService(
...
       if (otherSubscriptionsWithClient.isEmpty()) {
+        log.info("Deleting User Pool Client ID: ${subscription.oauthClientId}")
         cognitoService.deleteUserPoolClient(subscription.oauthClientId)
       }
     }
 
     if (subscription.queueName != null) {
       val otherSubscriptionsOnQueue = acquirerSubscriptionRepository.findAllByQueueName(subscription.queueName)
       if (otherSubscriptionsOnQueue.isEmpty()) {
+        val dqlQueueName = "${subscription.queueName}-dlq"
```
Comment (olevett) [inline src/main/kotlin/uk/gov/gdx/datashare/services/AcquirersService.kt:194]:
```suggestion
        val deadLetterQueueName = "${subscription.queueName}-dlq"
```

File: src/main/kotlin/uk/gov/gdx/datashare/services/AcquirersService.kt [hunk 2]
```diff
@@ -177,15 +183,18 @@ class AcquirersService(
...
+        log.info("Deleting queues: ${subscription.queueName} and $dqlQueueName")
         outboundEventQueueService.deleteQueue(subscription.queueName)
-        outboundEventQueueService.deleteQueue("${subscription.queueName}-dlq")
+        outboundEventQueueService.deleteQueue(dqlQueueName)
```
Comment (olevett) [inline src/main/kotlin/uk/gov/gdx/datashare/services/AcquirersService.kt:197]:
```suggestion
        outboundEventQueueService.deleteQueue(deadLetterQueueName)
```

File: src/main/kotlin/uk/gov/gdx/datashare/services/AcquirersService.kt [hunk 3]
```diff
@@ -177,15 +183,18 @@ class AcquirersService(
...
+        log.info("Deleting User Pool Client ID: ${subscription.oauthClientId}")
         cognitoService.deleteUserPoolClient(subscription.oauthClientId)
       }
     }
 
     if (subscription.queueName != null) {
       val otherSubscriptionsOnQueue = acquirerSubscriptionRepository.findAllByQueueName(subscription.queueName)
       if (otherSubscriptionsOnQueue.isEmpty()) {
+        val dqlQueueName = "${subscription.queueName}-dlq"
+        log.info("Deleting queues: ${subscription.queueName} and $dqlQueueName")
```
Comment (olevett) [inline src/main/kotlin/uk/gov/gdx/datashare/services/AcquirersService.kt:195]:
```suggestion
        log.info("Deleting queues: ${subscription.queueName} and $deadLetterQueueName")
```

File: src/main/kotlin/uk/gov/gdx/datashare/services/SuppliersService.kt [hunk 4]
```diff
@@ -107,6 +107,7 @@ class SuppliersService(
     )
     val otherSubscriptionsWithClient = supplierSubscriptionRepository.findAllByClientId(subscription.clientId)
     if (otherSubscriptionsWithClient.isEmpty()) {
+      AcquirersService.log.info("Deleting User Pool Client ID: ${subscription.clientId}")
```
Comment (olevett) [inline src/main/kotlin/uk/gov/gdx/datashare/services/SuppliersService.kt:110]: Should this be using the `AcquirersService` logger?

Comment (oskarwilliams) [reply src/main/kotlin/uk/gov/gdx/datashare/services/SuppliersService.kt:110]: I had hoped it would be smarter than this :(

**Review by olevett — APPROVED** [review]

**Pull request merged**